src/azure-cli/HISTORY.rst

```diff
@@ -3,9 +3,13 @@
 Release History
 ===============
 
-2.64.0
+2.65.0
 ++++++
 
+**MACHINE API**
+* Add `az aks machine list` command to fetch list of machines in an agentpool.
+* Add `az aks machine show` command to fetch information about a specific machine in an agentpool.
+
```

Comment:
Don't need to update the release history manually in official azure-cli. Instead, your PR title would be treated as the release note. Please update your PR title according to https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#format-pr-title